src/RadMarshakAsymptotic/test_radiation_marshak_asymptotic.cpp

```diff
@@ -326,7 +326,7 @@ auto problem_main() -> int
 		sol_norm += std::abs(Tmat_exact[i]);
 	}
 
-	const double error_tol = 0.05; // 5 per cent
+	const double error_tol = 0.09;
```

Comment:
This seems a bit large. Can you post a plot of the solution?

Reply:
Here is the new plot:
![marshak_wave_asymptotic_gastemperature](https://github.com/quokka-astro/quokka/assets/24463821/f359013d-e003-45e9-8841-87ff13769468)

compare to the old plot:
![marshak_wave_asymptotic_gastemperature-cfl0 9](https://github.com/quokka-astro/quokka/assets/24463821/8df59c03-e12b-4227-94ee-1049141bf4f9)

The new plot is smoother thanks to the new odd-even correction, but there is one single dot at x=0.14 that is ahead of the analytic solution by (maybe) 0.005, but the relative error is huge because of the sharp drop at the wave front. I think we can live with this.

Reply:
In cases like this is may just be that using |sim - exact| / exact as your definition of the error is not a good choice of how to define the error metric, since formally the exact solution is zero -- the only reason that the error didn't come out as infinite is that you have had to impose some finite temperature even in the upstream reason for numerical reasons. A better metric for error for this sort of problem might be to define the position of the front somehow (for example the first cell for which T > 1.1 * T_init) and check that the front position is accurate to a cell.

Reply:
We compare the total L1 norm of the error with the total L1 norm of the exact solution, sum(|sim - exact|) / sum(|exact|), so there is no problem of relative error being infinite. 

Indeed, I agree. We could add an extra metric to check the position of the wavefront.